### PR TITLE
Transaction to increase access node slots; Modify the signers on the service and staking account

### PR DESCRIPTION
**Pull request by vishalchangrani**
Transaction 1 to Increase access node slot count by 5.
Transaction 2 to do the following:
1. Remove FlowScan as a signer on the service and staking account.
2. Add Bluesign as a signer on the service and staking account (see [FLIP 208](https://github.com/onflow/flips/blob/main/governance/20231005-Adding-bluesign-as-multi-sig.md))

### CODE REVIEW

**Comment by Kay-Zee** [comment]
is the second tx not just a repeat of https://github.com/onflow/service-account/blob/main/templates/replace_key.cdc

File: transactions/add-access-nodes/2023/nov-16/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Increase Access node Slot Limit
+
+> Nov 16th, 2023 
+
+Increases the slot limit for Access nodes by 5
+
+## Using Multisign tool
+
+1. DapperLabs generates the Signature Request ID on the [site](https://flow-multisig-git-service-account-onflow.vercel.app/mainnet) for the `add_access_limits.cdc` transaction with the given args.
```
Comment (joshuahannan) [inline transactions/add-access-nodes/2023/nov-16/README.md:9]:
```suggestion
1. Flow/Vishal generates the Signature Request ID on the [site](https://flow-multisig-git-service-account-onflow.vercel.app/mainnet) for the `add_access_limits.cdc` transaction with the given args.
```
Probably should not mention Dapper Labs in these any more

File: transactions/add-keys/2023/nov-16/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+Key to be added to the service account and staking account:
+
+- `475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be` - Encoded (Sig: ECDSA_P256, Hash: SHA2_256, Weight: 250)
+- Index at which this key will get added
+  - Service account index: 18
+  - Staking account index: 10
+
+Key to be removed from the service account and the staking account:
+
+- `0xee4aec89e6d83fe563ee93de01374961fcf117b7e746c78b3097f54b630a053fd4ee70a4bde974ce12d71deb444c5c23c224ad92f05a6f72b54742fd9700a3b7`
```
Comment (joshuahannan) [inline transactions/add-keys/2023/nov-16/README.md:20]: The first key doesn't have a 0x, but the second one does. Shouldn't they be the same format?

File: transactions/add-keys/2023/nov-16/arguments_service_account.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "type": "Int",
+        "value": "17"
+    },
+    {
+        "type": "String",
+        "value": "f846b840ee4aec89e6d83fe563ee93de01374961fcf117b7e746c78b3097f54b630a053fd4ee70a4bde974ce12d71deb444c5c23c224ad92f05a6f72b54742fd9700a3b7020181fa"
```
Comment (Kay-Zee) [inline transactions/add-keys/2023/nov-16/arguments_service_account.json:8]: this encodes the old key, `ee4aec89e6d83fe563ee93de01374961fcf117b7e746c78b3097f54b630a053fd4ee70a4bde974ce12d71deb444c5c23c224ad92f05a6f72b54742fd9700a3b7`

from the readme, it sounds like you should be encoding `475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be`

**Review by joshuahannan — APPROVED** [review]
Can we update the `replace_keys` script to use `keys.add` and `keys.revoke` instead. The methods we are currently using are deprecated

File: transactions/add-keys/2023/nov-16/arguments_service_account.json [hunk 4]
```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "type": "Int",
+        "value": "17"
+    },
+    {
+        "type": "String",
+        "value": "475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be"
```
Comment (Kay-Zee) [inline transactions/add-keys/2023/nov-16/arguments_service_account.json:8]: this does not seem to be an encoded account key, which is what is expected by the tx, this is only the public key.

I believe the encoded key has to include the hash/sig algos and the weight:

`f846b840475aac6dc9e0a045323f8e24f0aa8ebbec9dfedc34a5cc47bbb206f8d639f03812231731a9608edab6582056ab0a3281350666330daca384c7dff777cc6823be020181fa`

**Review by KshitijChaudhary666 — APPROVED** [review]
Bluesign addition looks good. thanks

**Pull request merged**